phpcs.xml

```diff
@@ -4,6 +4,7 @@
 
     <rule ref="CakePHP"/>
     <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingNativeTypeHint">
+        <exclude-pattern>*/src/Command/*</exclude-pattern>
```

Comment:
You cannot do this

Reply:
Kenn to explain why? Seems to work fine in my apps.

Reply:
I guess as all methods would now be ignored. So maybe we just target `execute()`.